_includes/scripts.html

```diff
@@ -18,26 +18,43 @@
 <!-- Custom --->
 {% if page.handlebars == true %}
 <script src="{{ site.baseurl }}/assets/vendors/handlebars.js"></script>
-{% endif %}
-{% if page.usedatatables == true %}
-    <script src="{{ site.baseurl }}/assets/vendors/datatables/datatables.min.js"></script>
-    <script src="{{ site.baseurl }}/assets/datatables-it.js"></script>
-{% endif %}
-{% if page.ref == 'homepage' or page.ref == 'dashboard' %}
-    <link href="{{ site.baseurl }}/assets/dashboard/app/js/app.js" rel="preload" as="script">
-    <link href="{{ site.baseurl }}/assets/dashboard/app/js/chunk-vendors.js" rel="preload" as="script">
-
-    <script src="{{ site.baseurl }}/assets/dashboard/app/js/chunk-vendors.js"></script>
-    <script src="{{ site.baseurl }}/assets/dashboard/app/js/app.js"></script>
-{% endif %}
-
-{% if page.ref == 'cittadini-trasparenza-costi' %}
-    <script src="{{ site.baseurl }}/assets/trasparenza-costi.js"></script>
-{% endif %}
-
-{% if page.js %}
-    {% for script in page.js %}
-    <script src="{{ site.baseurl }}{{script.url}}" charset="UTF-8"
-        {% if script.id %}id="{{script.id}}" {% endif %} ></script>
-    {% endfor %}
-{% endif %}
+{% endif %} {% if page.usedatatables == true %}
+<script src="{{ site.baseurl }}/assets/vendors/datatables/datatables.min.js"></script>
+<script src="{{ site.baseurl }}/assets/datatables-it.js"></script>
+{% endif %} {% if page.ref == 'homepage' or page.ref == 'dashboard' %}
+<script
+  defer=""
+  crossorigin=""
+  nomodule=""
+  src="{{ site.baseurl }}/assets/dashboard/_next/static/chunks/polyfills.js"
+></script>
+<script
+  src="{{ site.baseurl }}/assets/dashboard/_next/static/chunks/webpack.js"
+  defer=""
+  crossorigin=""
+></script>
+<script
+  src="{{ site.baseurl }}/assets/dashboard/_next/static/chunks/main.js"
+  defer=""
+  crossorigin=""
+></script>
+<script
+  src="{{ site.baseurl }}/assets/dashboard/_next/static/chunks/pages/_app.js"
+  defer=""
+  crossorigin=""
+></script>
+{% endif %} {% if page.ref == 'cittadini-trasparenza-costi' %}
+<script src="{{ site.baseurl }}/assets/trasparenza-costi.js"></script>
+{% endif %} {% if page.js %} {% for script in page.js %}
+<script
+  src="{{ site.baseurl }}{{script.url}}"
+  charset="UTF-8"
+  {%
+  if
+  script.id
+  %}id="{{script.id}}"
+  {%
+  endif
+  %}
+></script>
+{% endfor %} {% endif %}
```

Comment:
The actual changes are between rows 25 and 45. Code reformatting and missing line feeds have impacted the others (1-24 and 46-60). Please reinstate the initial condition.

```suggestion
    <script defer="" crossorigin="" nomodule="" src="{{ site.baseurl }}/assets/dashboard/_next/static/chunks/polyfills.js"></script>
    <script src="{{ site.baseurl }}/assets/dashboard/_next/static/chunks/webpack.js" defer="" crossorigin=""></script>
    <script src="{{ site.baseurl }}/assets/dashboard/_next/static/chunks/main.js" defer="" crossorigin=""></script>
    <script src="{{ site.baseurl }}/assets/dashboard/_next/static/chunks/pages/_app.js" defer="" crossorigin=""></script>
```